Project.toml

```
@@ -3,11 +3,12 @@ uuid = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 keywords = ["markov chain monte carlo", "probablistic programming"]
 license = "MIT"
 desc = "A lightweight interface for common MCMC methods."
-version = "2.5.0"
+version = "2.5.1"
```

This is a new feature, not a bugfix, hence we should bump the minor version:
```suggestion
version = "2.6.0"
```